docs/troubleshooting-guide/application-database.md

```diff
@@ -0,0 +1,54 @@
+## Specific Problems:
+
+
+### Metabase fails to start due to database locks
+
+Sometimes Metabase will fail to complete its startup due to a database lock that was not cleared properly. The error message will look something like:
+
+    liquibase.exception.DatabaseException: liquibase.exception.LockException: Could not acquire change log lock.
+
+When this happens, go to a terminal where Metabase is installed and run:
+
+    java -jar metabase.jar migrate release-locks
+
+in the command line to manually clear the locks. Then restart your Metabase instance.
+
+### Metabase H2 application database gets corrupted
+
+Because H2 is an on-disk database, it is sensitive to filesystem errors. Sometimes drives get corrupted, or the file doesn't get flushed correctly, which can result in a corrupted database. In these situations, you'll see errors on startup. These vary, but one example is 
+```
+myUser@myIp:~$ java -cp metabase.jar org.h2.tools.RunScript -script whatever.sql -url jdbc:h2:~/metabase.db
+Exception in thread "main" org.h2.jdbc.JdbcSQLException: Row not found when trying to delete from index """"".I37: ( /* key:7864 */ X'5256470012572027c82fc5d2bfb855264ab45f8fec4cf48b0620ccad281d2fe4', 165)" [90112-194]
+    at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
+    [etc]
+```
+
+Not all H2 errors are recoverable (which is why if you're using H2, _please_ have a backup strategy for the application database file). To attempt to recover a corrupted H2 file, try the below.
+
+```
+java -cp metabase.jar org.h2.tools.Recover
+mv metabase.db.mv.db metabase.old.db
+touch metabase.db.mv.db
+java -cp metabase.jar org.h2.tools.RunScript -script metabase.db.h2.sql -url jdbc:h2:/path/to/metabase.db
```

Comment:
nope. It is `metabase.db.h2.sql`. So that's good. I do have one improvement to this this script tho from testing it.

Change the last line to:

```bash
java -cp target/uberjar/metabase.jar org.h2.tools.RunScript -script metabase.db.h2.sql -url jdbc:h2:`pwd`/metabase.db
```

should make it more foolproof.

We can also do the same to the one below, just make sure to tack on `MV_STORE=FALSE`